### PR TITLE
Protect installer service modifying methods with permissions

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Protects installer service modifying methods (add, update, remoce) with manage system permissions.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/8370

#### Release Notes
Protected installer service modifying methods with permissions

#### Docs PR
N/A

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

File: multiuser/permission/che-multiuser-permission-installer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-multiuser-permission</artifactId>
+        <groupId>org.eclipse.che.multiuser</groupId>
+        <version>6.1.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>che-multiuser-permission-installer</artifactId>
+    <name>Che Multiuser :: Installer Permissions</name>
+    <properties>
+        <findbugs.failonerror>false</findbugs.failonerror>
```
Comment (sleshchenko) [inline multiuser/permission/che-multiuser-permission-installer/pom.xml]: Can we enable fail on an error in this module?

**Review by sleshchenko — APPROVED** [review]
LGTM

**Review by skabashnyuk — APPROVED** [review]

**Review by artaleks9 — APPROVED** [review]
Test report is OK

**Pull request merged**